Code/Tools/Libs/GuiFoundation/UIServices/DynamicBitflags.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <Foundation/Containers/Map.h>
+#include <Foundation/Strings/String.h>
+#include <GuiFoundation/GuiFoundationDLL.h>
+
+/// \brief Stores the valid values and names for 'dynamic' bitflags.
+///
+/// The names and valid values for dynamic bitflags may change due to user configuration changes.
+/// The UI should show these user specified names without restarting the tool.
+///
+/// Call the static function GetDynamicBitflags() to create or get the ezDynamicBitflags for a specific type.
+class EZ_GUIFOUNDATION_DLL ezDynamicBitflags
+{
+public:
+  /// \brief Returns a ezDynamicBitflags under the given name. Creates a new one, if the name has not been used before.
+  static ezDynamicBitflags& GetDynamicBitflags(const char* szName);
```

Comment:
For new code please prefer ezStringView (I know this was copied from existing code).

Reply:
Yes that is correct. I can make the change to ezStringView.